model/database/registry.yaml

```diff
@@ -127,238 +127,181 @@ groups:
+              brief: "[Amazon DynamoDB](https://aws.amazon.com/pm/dynamodb/)"
+              stability: development
+            - id: aws.redshift
+              value: "aws.redshift"
+              brief: "Amazon Redshift"
+              stability: development  # trademark on 'amazon redshift'
+            - id: az.cosmosdb
+              value: "az.cosmosdb"  # trademark on 'azure cosmos db'
+              brief: "[Microsoft Azure Cosmos DB](https://learn.microsoft.com/azure/cosmos-db)"
+              stability: development
+            - id: intersystems.cache
+              value: "intersystems.cache"  # trademark on 'INTERSYSTEMS CACHÉ'
+              brief: "[InterSystems Caché](https://www.intersystems.com/products/cache/)"
+              stability: development
             - id: cassandra
-              value: "cassandra"
-              brief: "Apache Cassandra"
-              stability: experimental
+              value: "cassandra"  # trademark on 'cassandra'
+              brief: "[Apache Cassandra](https://cassandra.apache.org/)"
+              stability: development
             - id: clickhouse
-              value: "clickhouse"
-              brief: "ClickHouse"
-              stability: experimental
-            - id: cloudscape
-              value: "cloudscape"
-              deprecated: "Replaced by `other_sql`."
-              brief: "Deprecated, use `other_sql` instead."
-              stability: experimental
+              value: "clickhouse"  # no wordmark
+              brief: "[ClickHouse](https://clickhouse.com/)"
+              stability: development
             - id: cockroachdb
-              value: "cockroachdb"
-              brief: "CockroachDB"
-              stability: experimental
-            - id: coldfusion
-              value: "coldfusion"
-              deprecated: "Removed."
-              brief: "Deprecated, no replacement at this time."
-              stability: experimental
-            - id: cosmosdb
-              value: "cosmosdb"
-              brief: "Microsoft Azure Cosmos DB"
-              stability: experimental
+              value: "cockroachdb"  # trademark on 'cockroachdb'
+              brief: "[CockroachDB](https://www.cockroachlabs.com/)"
+              stability: development
             - id: couchbase
-              value: "couchbase"
-              brief: "Couchbase"
-              stability: experimental
+              value: "couchbase"  # trademark on 'couchbase'
+              brief: "[Couchbase](https://www.couchbase.com/)"
+              stability: development
             - id: couchdb
-              value: "couchdb"
-              brief: "CouchDB"
-              stability: experimental
-            - id: db2
-              value: "db2"
-              brief: "IBM Db2"
-              stability: experimental
+              value: "couchdb"  # trademark on 'couchdb'
+              brief: "[Apache CouchDB](https://couchdb.apache.org/)"
```

Comment:
I keep checking the official names to see if the name has `DB` together like this one, or separate like Cosmos... why can't it be consistent? 😅